Controllers/HomeController.cs

```diff
@@ -3,21 +3,90 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using System.Text.RegularExpressions;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
 
 namespace Zoltu.Bags.Client.Controllers
 {
 	public class HomeController : Controller
 	{
+		// Best practices
+		// https://www.asp.net/web-api/overview/advanced/calling-a-web-api-from-a-net-client
+		static HttpClient client = new HttpClient();
+
 		[Route("")]
 		[Route("app/{*path}")]
-		public IActionResult App(string path = null)
+		public IActionResult App(String path = null)
```

Comment:
Performance: Now that we are calling an external service, this call stack should be changed to an async one.  This method and `CallApi` should be like `public async Task<IActionResult> App(String path = null)` and when you call one of them, call like `await CallApi(...)`